legacyCore/src/main/res/values/dimens.xml

```diff
@@ -1,14 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Todo remove these resource files / move them to correct location FA-100 -->
 <resources>
+    <dimen name="strokeSmall1">1dp</dimen>
+    <dimen name="strokeSmall2">2dp</dimen>
+    <dimen name="strokeMed1">3dp</dimen>
+    <dimen name="strokeMed2">4dp</dimen>
+    <dimen name="strokeLarge1">5dp</dimen>
+    <dimen name="strokeLarge2">6dp</dimen>
+
     <dimen name="buttonSmall1">40dp</dimen>
```

Comment:
Are these applied on buttons? Buttons should be the size they're going to be with text size + padding. Enforcing a rigid size can lead to clipping

Reply:
yes they were button sizes, for the buttons they work fine because theres no internal padding on the buttons, however useing these to match text field sizes does not work, so I have removed strict sizes to the text field for the height